ADR/0032-managing-secrets-in-managed-workspaces.md

```diff
@@ -0,0 +1,14 @@
+# 32. Managing Secrets in Managed Workspaces
+Date: 2023-11-22
+
+## Status
+Accepted
+
+## Context
+Currently, secrets are managed in managed workspaces via SPI's RemoteSecrets. This involves creating RemoteSecret resources in addition to creating and uploading Secrets to be linked to the RemoteSecrets. This secret data does not exist in any gitops repos nor within any secret service provider. It only exists locally on one's laptop. This poses a problem when a service or team needs to rotate a secret or even verify if the secret is valid.
```

Comment:
> This secret data does not exist in any gitops repos nor within any secret service provider.

Is it considered that this data will be pre-populated (apparently by dedicated person, like system admin) or Managed workspaces consider some self-servicing in this regard? 
If latter it become more interesting b/c ESO is likely designed for the first case.

Reply:
@gazarenkov , do you need this question answered still?